core/common/src/test/java/alluxio/ConfigurationTest.java

```diff
@@ -507,4 +507,22 @@ public void sitePropertiesLoadedNotInTest() throws Exception {
       assertEquals("TEST_LOGGER", Configuration.get(PropertyKey.LOGGER_TYPE));
     }
   }
+
+  @Test
+  public void discardIgnoredSitePropertiesTest() throws Exception {
+    Properties siteProps = new Properties();
+    siteProps.setProperty(PropertyKey.MASTER_HOSTNAME.toString(), "host-1");
+    siteProps.setProperty(PropertyKey.LOGS_DIR.toString(), "/tmp/logs1");
+    File propsFile = mFolder.newFile(Constants.SITE_PROPERTIES);
+    siteProps.store(new FileOutputStream(propsFile), "tmp site properties file");
+    Map<String, String> sysProps = new HashMap<>();
+    sysProps.put(PropertyKey.LOGS_DIR.toString(), "/tmp/logs2");
```

Comment:
I think this test would pass even before your change because system properties override site properties. Make sure the test fails before your changes, otherwise it isn't really testing them. We can fix the issue by not using system properties in this test at all - the test is supposed to be about discarding site properties anyway. We can use a second, simpler test to test that `ignoredSiteProperty` keys still work as JVM properties

Reply:
Done.